src/shadowbox/server/manager_service.spec.ts

```diff
@@ -258,10 +258,22 @@ describe('ShadowsocksManagerService', () => {
         send: (httpCode, data) => {
           expect(httpCode).toEqual(201);
           expect(Object.keys(data).sort()).toEqual(EXPECTED_ACCESS_KEY_PROPERTIES);
+          expect(data.method).toEqual('chacha20-ietf-poly1305');
           responseProcessed = true;  // required for afterEach to pass.
         }
       };
-      service.createNewAccessKey({params: {}}, res, done);
+      service.createNewAccessKey({params: {encryptionMethod: 'chacha20-ietf-poly1305'}}, res, done);
```

Comment:
- Revert this and make the test assert that the default cipher is chacha...,
- Add a new test that takes a different encryption method and make sure that's set.
- Add a test that verified that a banned encryption method fails.

Reply:
* done
* done 
* working on the banned encryption

Reply:
done.